lib/resolvers/aix/networking_resolver.rb

```diff
@@ -27,6 +30,47 @@ def get_primary_interface_info(output)
             @fact_list[:primary] = primary_interface_info[5]
             @fact_list[:ip] = primary_interface_info[1]
           end
+
+          def extract_interfaces(netstat_output)
+            netstat_output.each do |line|
+              next if line =~ /default/
+
+              info = line.split("\s")
+              mask_length = info[0].match(%r{/([0-9]+)|%([0-9]+)})
+              next unless mask_length
+
+              build_bindings(info[5], info[1], mask_length[1] || mask_length[2],
+                             info[1] =~ /[0-9]+.[0-9]+.[0-9]+.[0-9]+/)
+            end
+
+            ::Resolvers::Utils::Networking.expand_main_bindings(@fact_list[:interfaces])
+          end
+
+          def build_bindings(name, ip, mask_length, is_ipv4)
+            bind_to_add = is_ipv4 ? :bindings : :bindings6
+            ip = ip.gsub(/%[0-9]$/, '') # remove mask information if it exists
+            mask_length = mask_length.to_i - 1 unless is_ipv4
+            @fact_list[:interfaces][name] ||= {}
+            @fact_list[:interfaces][name][bind_to_add] ||= []
+            @fact_list[:interfaces][name][bind_to_add] << ::Resolvers::Utils::Networking.build_binding(ip, mask_length)
+          end
+
+          def populate_with_mtu_and_mac!(interfaces)
+            output = Facter::Core::Execution.execute('netstat -in', logger: log)
+            output.split("\n").each do |line|
```

Comment:
`split("\n")` can be replaced with `each_line`

Reply:
Done